src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -156,263 +236,161 @@ users, possibly hurting their overall experience.
-| Differential indicators           | Useful when users are monitoring differentials in large lists of statistics and when anything other than type would be too obtrusive.                                     | Most often used in financial dashboards for highlighting deltas or other types of data visualizations.                                                                                                                          |
+| Variant         | Icon size | Type size |
+| --------------- | --------- | --------- |
+| Icon indicator  | 20px      | 16pt      |
+|                 | 16px      | 14pt      |
+| Shape indicator | 16px      | 14pt      |
+|                 | 16px      | 12pt      |
 
-### Anatomy
-
-Like a button, a status indicator’s text label is the most important element, as
-it communicates progress. By default, Carbon uses sentence case for all button
-labels.
+## Icon indicator
 
-Text labels, column labels (or a legend, when inline labels aren’t an option)
-are strongly recommended.
+Icon indicators consist of an icon, a shape, a meaningful color, and a
+descriptive label. They are highly effective for communication and enhance
+scannability, especially in content-heavy layouts. Common uses include
+notifications, progress indicators, data tables, task lists, and dashboard
+widgets.
 
 <Row>
 <Column colLg={8}>
 
-![Anatomy images of the five types of status indicators](./images/status_indicator_3.png)
-
-</Column>
-</Row>
-
-<Row>
-<Column colSm={2} colMd={4} colLg={4}>
+![Carbon notification component](./images/status-icon-indicator.png)
 
-#### 1. Icon indicator
-
-A. Text label <br /> B. Symbol <br /> C. Shape <br /> D. Color
-
-#### 3. Badge indicator (no number)
-
-A. Shape without number <br /> B. Symbol <br /> C. Color
-
-#### 5. Differential indicator
-
-A. Text label <br /> B. Symbol\* <br /> C. Color (optional)
+<Caption>
+  Notifications are the most prevalent example of this type of status indicator.
+</Caption>
 
 </Column>
-<Column colSm={2} colMd={4} colLg={4}>
-
-#### 2. Badge indicator (number)
-
-A. Shape with number <br /> B. Icon <br /> C. Color
-
-#### 4. Shape indicator
+</Row>
 
-A. Text label <br /> B. Shape <br /> C. Color
+### Icon indicator statuses
 
-</Column>
-</Row>
+Suggested icon indicator status names are based on common use cases across
+products to help guide you in defining status notifications according to your
+system’s needs. These names are not intended to dictate the final status names
+in your product.
 
-<Caption>
-  *Differential indicators must have either a ‘+’ or ‘-’ sign, a caret, or an
-  arrow icon to indicate positive or negative values.
-</Caption>
+<StatusIndicatorTable attention="high" />
```

Comment:
Since we are showing color tokens now instead of color values, I think we should make them inline code snippets like we do on the style tabs. This should also be done in the Shape indicator statuses table.